boards/arm/arduino_uno_r4/arduino_uno_r4_minima.dts

```diff
@@ -32,6 +32,35 @@
 	aliases {
 		led0 = &led;
 	};
+
+	arduino_header: connector {
```

Comment:
The convention (although not a rule afaik) would be to have this in a separate .dtsi file.
You should also define the aliases such as `&arduino_i2c`

Reply:
Ok that makes sense, thanks!